src/showcase/src/layouts/Screen.astro

```diff
@@ -0,0 +1,19 @@
+---
+import Layout from "../layouts/Layout.astro";
+
+interface Props {
```

Comment:
nit: we use `type` pretty much everywhere in the codebase

Reply:
Sounds good